tools/serverpod_cli/lib/src/generator/open_api/objects/security.dart

```diff
@@ -0,0 +1,223 @@
+part of '../open_api_objects.dart';
+
+/// Defines a security scheme that can be used by the operations.
```

Comment:
What operations are we referring to? A clarification in the comment would be nice.